examples/ErrorHandling/HandleRateExceededError.php

```diff
@@ -0,0 +1,263 @@
+            ->build();
+
+        try {
+            self::runExample(
+                $googleAdsClient,
+                $options[ArgumentNames::CUSTOMER_ID] ?: self::CUSTOMER_ID,
+                $options[ArgumentNames::AD_GROUP_ID] ?: self::AD_GROUP_ID
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $adGroupId the ad group ID to validate keywords from
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $adGroupId
+    ) {
+        // Sequentially sends the requests.
+        for ($i = 0; $i < self::NUM_REQUESTS; $i++) {
+            // Creates operations.
+            $operations = self::createAdGroupCriterionOperations($customerId, $adGroupId, $i);
+
+            try {
+                $retryCount = 0;
+                $retrySeconds = self::RETRY_SECONDS;
+                while ($retryCount < self::NUM_RETRIES) {
+                    try {
+                        // Sends request.
+                        self::requestMutateAndDisplayResult(
+                            $googleAdsClient,
+                            $customerId,
+                            $operations
+                        );
+                        break;
+                    } catch (GoogleAdsException $googleAdsException) {
+                        foreach ($googleAdsException->getGoogleAdsFailure()
+                                     ->getErrors() as $googleAdsError) {
+                            // Checks if any of the errors are QuotaError.RESOURCE_EXHAUSTED or
+                            // QuotaError.RESOURCE_TEMPORARILY_EXHAUSTED.
+                            if ($googleAdsError->getErrorCode()->getQuotaError() ==
+                                QuotaError::RESOURCE_EXHAUSTED ||
+                                $googleAdsError->getErrorCode()->getQuotaError() ==
+                                QuotaError::RESOURCE_TEMPORARILY_EXHAUSTED) {
```

Comment:
Nit: we usually follow Google Java style by having an operator in the beginning of a line, e.g.,

== blah blah
|| blah blah